provider/pkg/clients/clients.go

```diff
@@ -257,6 +264,25 @@ func IsCRD(obj *unstructured.Unstructured) bool {
 		strings.HasPrefix(obj.GetAPIVersion(), "apiextensions.k8s.io/")
 }
 
+// FindCRD finds the CRD for a given kind amongst a list of unstructured objects.
+func FindCRD(objs []unstructured.Unstructured, kind schema.GroupKind) (*unstructured.Unstructured, bool) {
```

Comment:
Looks like we don't use the returned bool value, should we simplify this function's return signture?